firefox-ios/firefox-ios-tests/Tests/ClientTests/NativeErrorPage/Mock/NativeErrorPageMockModel.swift

```
@@ -9,7 +9,9 @@ final class NativeErrorPageMock {
     static var model: ErrorPageModel {
```

I'd remove this class since in favor of adding a property in the interested test class who just set mock data for the `ErrorPageModel`, what do you think ?

---

Good suggestion. I will remove mock error page model.

---

Thanks @tusharC95